### PR TITLE
update regex in AccessLogger to accept underscore

### PR DESCRIPTION
**Pull request by esaezgil**
## What do these changes do?

Update regex in AccessLogger to accept underscore, update Unit Tests
## Are there changes in behavior for the user?
no

## Related issue number
#1225 

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] Add yourself to `CONTRIBUTORS.txt`
- [x] Add a new entry to `CHANGES.rst`
  * Choose any open position to avoid merge conflicts with other PRs.
  * Add a link to the issue you are fixing (if any) using `#isuue_number` format at the end of changelog message. Use Pull Request number if there are no issues for PR or PR covers the issue only partially.

update Unit Tests

### CODE REVIEW

File: aiohttp/helpers.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ class AccessLogger:
     """
 
     LOG_FORMAT = '%a %l %u %t "%r" %s %b "%{Referrer}i" "%{User-Agent}i"'
-    FORMAT_RE = re.compile(r'%(\{([A-Za-z\-]+)\}([ioe])|[atPrsbOD]|Tf?)')
+    FORMAT_RE = re.compile(r'%(\{([A-Za-z\-_]+)\}([ioe])|[atPrsbOD]|Tf?)')
```
Comment (popravich) [inline aiohttp/helpers.py]: maybe numbers should be accepted as well?

Comment (asvetlov) [reply aiohttp/helpers.py]: agree

Comment (asvetlov) [reply aiohttp/helpers.py]: Not sure should we split header rules and environment variable

Comment (esaezgil) [reply aiohttp/helpers.py]: updated to accept numbers, not sure about your last comment though

File: CHANGES.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ CHANGES
 1.0.3 (XXXX-XX-XX)
 ------------------
 
--
+- Modify regex in AccessLogger to accept underscore #1225
```
Comment (asvetlov) [inline CHANGES.rst]: Should be in 1.1.0 section unless you make a PR against 1.0 branch.

File: aiohttp/helpers.py (hunk 1 above)
Comment (asvetlov) [reply aiohttp/helpers.py]: good to me

File: CHANGES.rst (hunk 2 above)
Comment (esaezgil) [reply CHANGES.rst]: Thanks for pointing this out, updated

**Comment by esaezgil** [comment]
I am already in CONTRIBUTORS.txt from a previous contribution :)

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**